lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :dishes, :report_price
+
+# should not be hard-coded!
+  DEFAULT_MENU = { 'Coffee' => 2.50, 'Tea' => 2.00, 'Beer' => 3.50, 'Wine' => 4.00 }
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)